ui/app/services/path-help.js

```diff
@@ -204,10 +204,18 @@ export default Service.extend({
         };
       }
 
-      // TODO: handle post endpoints without requestBody
-      const props = pathInfo.post
-        ? pathInfo.post.requestBody.content['application/json'].schema.properties
-        : {};
+      let props = {};
+      const schema = pathInfo?.post?.requestBody?.content['application/json'].schema;
+      if (schema.$ref) {
+        // $ref will be shaped like `#/component/schema/MyResponseType
```

Comment:
super small comment but components and schemas are plural in the `$ref` 
` "$ref": "#/components/schemas/KvLookupRequest"`

...but does that affect your mapping?

Reply:
oh nope! it doesn't, since you're using `reduce` and setting the keys on 212

Reply:
no, but I should probably be accurate in my comment 😄